css/layout/layout.css

```diff
@@ -0,0 +1,25 @@
+.header__main {
```

Comment:
the layout mindset is good, but, a header is more of a component than a layout, but (again) on the other hand, if you can place content on top of your layout class then you good to go with a layout, a good example of this is css grid.